### PR TITLE
Add lightstreamer-server documentation

### PR DESCRIPTION
**Pull request by gfinocchiaro**

### CODE REVIEW

**Comment by gfinocchiaro** [comment]
Don't understand while the build failed.

**Comment by gfinocchiaro** [comment]
Any news on this PR?

Thanks

**Comment by gfinocchiaro** [comment]
Is there any chance to go on with this PR?

Thanks

File: update.sh [hunk 1]
```diff
@@ -47,6 +47,7 @@ declare -A otherRepos=(
 	[joomla]='https://github.com/joomla/docker-joomla'
 	[jruby]='https://github.com/cpuguy83/docker-jruby'
 	[kaazing-gateway]='https://github.com/kaazing/gateway.docker'
+        [lightstreamer-server]='https://github.com/Lightstreamer/Docker'
```
Comment (tianon) [inline update.sh]: Can you replace these spaces with a tab?

**Comment by tianon** [comment]
LGTM sans the minor whitespace issue in `update.sh`

**Comment by tianon** [comment]
(if you can squash the commits too, that'd be great :+1:)

**Comment by gfinocchiaro** [comment]
Just done! Let me know.

Thanks!

**Comment by yosifkit** [comment]
Just noticed that this needs an update for the folder and addition to `update.sh` to be `lightstreamer`, rather than `lightstreamer-server`.

Otherwise, LGTM.

File: lightstreamer-server/content.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+%%LOGO%%
+
+# How to use this image
+
+## Up and Running
+
+Launch the container with the default configuration:
+
+```console
+$ docker run --name ls-server -d -p 80:8080 lightstreamer-server
```
Comment (yosifkit) [inline lightstreamer-server/content.md]: don't forget any `lightstreamer-server` in the content, like this one :wink:

**Comment by gfinocchiaro** [comment]
Oh! Done! Thanks a lot for this review!

**Comment by yosifkit** [comment]
Nice, LGTM.

**Comment by gfinocchiaro** [comment]
Great! Thank you very much!
I noticed that in the explore page at hub.docker.com, there is no icon next to the link to official page.
How to make it appear?

**Comment by psftw** [comment]
@gfinocchiaro :+1: I pinged the team responsible for those logos with a handful of missing ones today, so hopefully they'll be added soon.  Sorry for the delay.

**Comment by gfinocchiaro** [comment]
Thank you very much @psftw!

**Pull request merged**